### PR TITLE
Add in s390x build

### LINKED ISSUE #2458: Support s390x Architecture
**Is your feature request related to a problem? Please describe.**
As far as I can tell, the External Secrets Operator does not support the s390x architecture. As far as I can tell it currently only supports amd64 and arm64.

**Describe the solution you'd like**
I would like External Secrets Operator to support s390x architecture.

**Describe alternatives you've considered**
I've considered building the image myself but that option may not be available to everyone or some may prefer an official image.

**Additional context**
NA

### PR DESCRIPTION
**Pull request by Joshua-Beha**
## Problem Statement

Currently, the External Secrets operator does not build images for the s390x architecture. 

## Related Issue

Fixes #2458 

## Proposed Changes

I am solving this issue by adding a new tag to the build automation that will include an s390x build using the provided `Dockerfile.standalone` file in the root of this repo.

## Checklist

- [ x ] I have read the [contribution guidelines](https://external-secrets.io/latest/contributing/process/#submitting-a-pull-request)
- [ x ] All commits are signed with `git commit --signoff`
- [ x ] My changes have reasonable test coverage
- [ x ] All tests pass with `make test`
- [ x ] I ensured my PR is ready for review with `make reviewable`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -180,6 +180,11 @@ jobs:
           build-arch: "amd64"
           build-platform: "linux/amd64"
           tag-suffix: "-ubi-boringssl"
+        - dockerfile: "Dockerfile.standalone"
+          build-args: "CGO_ENABLED=0"
+          build-arch: "amd64 arm64 s390x"
+          build-platform: "linux/amd64,linux/arm64,linux/s390x"
+          tag-suffix: "multiarch" # build for multiarch
```
Comment (moolen) [inline .github/workflows/ci.yml:187]: why do we want to use the `Dockerfile.standalone` here and provide a separate tag? 
I think we can just add  `s390x` to the first list element above, there is a multiarch build for it already, i think this should work. WDYT?

Comment (Joshua-Beha) [reply .github/workflows/ci.yml]: Oh, I see what should be happening now. Just made the updates :)

**Comment by moolen** [comment]
Hey, can you please rebase? I think something went wrong with that merge

**Comment by Joshua-Beha** [comment]
Should be set now, I didn't have some of the commits signed correctly.

**Review by moolen — APPROVED** [review]
LGTM, image gets built, need to merge to main to build/inspect the image there.

Thanks for adding IBM zSystems and LinuxONE support :bow:

**Pull request merged**